### ISSUE TITLE
SQL data race (1.8.1-586-g6e8913e)

### ISSUE THREAD

**Issue by ctpo6** [tarantool/tarantool#2723]
Перетирание соседних строк таблицы одновременными UPDATE.

[Скрипт](https://pastebin.com/WY9EmtQG)

**Comment by kyukhin** [comment]
Confirmed.

**Comment by Khatskevich** [comment]
blocked by #2741

**Comment by Khatskevich** [comment]
I have created a new issue for concurrent ddls: #2793
For simple queries (dml), concurrent executions work pretty fine.
